src/components/Characters.js

```diff
@@ -14,19 +14,42 @@ const getCharacters = async () => {
   return characters.persons;
 }
 
+const getNpcs = async () => {
+    const response = await fetch(apiUrl("/person?show_hidden=true&is_character=false"));
+    const npcs = await response.json();
+    return npcs.persons;
+  }
+
 export default function Characters(props) {
   const [characters, setCharacters] = React.useState([]);
   const [page, setPage] = React.useState(1);
   const [sizePerPage, setSizePerPage] = React.useState(15);
 
   React.useEffect(() => {
-    getCharacters().then(data => setCharacters(data));
+    const fetchData = async () => {
+      try {
+        const charactersData = await getCharacters();
+        const npcsData = await getNpcs();
```

Comment:
No need to change this, but these two could be optimized to run at the same time:
```suggestion
        const [charactersData, npcsData] = await Promise.all([
            getCharacters(),
            getNpcs(),
        ]);
```